### ISSUE TITLE
Function "Display Name" administration page

### ISSUE THREAD

**Issue by Erriez** [Mailu/Mailu#2100]
## Before you open your issue
- [X] Check if no issue or pull-request for this already exists.
- [X] Check [documentation](https://mailu.io/master/) and [FAQ](https://mailu.io/master/faq.html). (Tip, use the search function on the documentation page)
- [X] You understand `Mailu` is made by volunteers in their **free time** — be conscise, civil and accept that delays can occur.
- [X] The title of the issue should be short and simple. It should contain specific terms related to the actual issue. Be specific while writing the title.

### Environment
 - [X] docker-compose

### Versions

`master hash 18865bf Tue Dec 21 13:08:48 2021`

## Description
The [master Display Name](https://mailu.io/master/webadministration.html#display-name) documentation describes:

> On the settings page the displayed name can be changed of the logged in user. This display name is only used within the web administration interface.

At which locations should the `Displayed name` shown? When I login, I see the email address in the left bar and not display name.

![image](https://user-images.githubusercontent.com/8849873/147085991-961ca0b0-0f0e-4ddc-b1b1-c4098fbfdced.png)

## Replication Steps
1. Login to Administration page
2. Change `Displayed name` and click `Save`
3. The user icon shows the email address and not filled in Displayed Name.

Thanks!

**Comment by Diman0** [comment]
That used to be the behaviour in Mailu 1.7. This must have been changed in 1.8 ( I see the same in 1.8).

**Comment by Diman0** [comment]
No that is not correct. Sidebar.html did not use the display name in the 1.7 branch.
https://github.com/Mailu/Mailu/search?q=displayed_name
Looking in the project it is only used in the vacation (out of office) functionality
https://github.com/Mailu/Mailu/blob/3ccb6ff4b5f816724de71489a5865074396d6f3e/core/admin/mailu/internal/templates/default.sieve#L36
It is used in the "from: ..." part for auto-reply emails.

The documentation should be updated to reflect this.

**Comment by Erriez** [comment]
@Diman0 Thanks for your explanation.

Would it be possible to use the `Displayed name` field in the email clients as well? Finally I found the setting in `Rainloop | Settings  |General | Identity | Name`, because it displays the email address without the name in the `To:`.

**Comment by Diman0** [comment]
The webmail clients are imap clients. This information (display name) is not accessible via IMAP. 

A custom plugin would need to be written for each webmail to retrieve the display name from Mailu Admin. In Mailu Admin a new API must be added to retrieve the username of a user. I don't think it is likely someone will create this.

Rainloop stores all data in your internet browser's storage. So the configured display name is only available in your internet browser until the local browser storage is cleared.

In roundcube all data is stored in a database. So the settings are persistent. To change the setting in roundcube go to (in roundcube) Settings -> Identities -> Your user -> display name.

BTW I strongly recommend to use roundcube. We just discovered that Rainloop has multiple security issues which haven not been patched for years. In the future we are going to look for a replacement for rainloop.

**Comment by Erriez** [comment]
As always: Thanks for your clear explanation. Good to know how this is internally handled and saves me a lot of time. 

The current behavior of `Displayed name` is fine for me, as long as it is documented somewhere. Maybe a message can be displayed when hovering over the text field that this name is only used for out of office replies (I confirm that this works), but I don't know how to translate such message into many languages. (Google Translate mabye?)

 I can create PR to update `docs/webadministration.rst` for example to:

```diff
Display name

On the settings page the displayed name can be changed of the logged in user.
-This display name is only used within the web administration interface.
+This display name is only used within the web administration interface and sending automatic email replies (out of office replies).
```

> In the future we are going to look for a replacement for rainloop.

Good to know this. I used Rainloop for many years and I like simplicity / lightweight. I used [Snappymail.eu](https://snappymail.eu/) as well which is a fork of Rainloop. It looks like better maintained and I noticed some commits today. A Docker container is available, so maybe it is an option to consider.

However, I don't want to introduce more work, so I'll switch to Roundcube now as you suggested.
